internal/provider/data_source_workspace_test.go

```diff
@@ -222,6 +222,8 @@ func TestAccTFEWorkspaceDataSource_readAutoDestroyAt(t *testing.T) {
 }
 
 func TestAccTFEWorkspaceDataSource_readAutoDestroyDuration(t *testing.T) {
+	skipUnlessAfterDate(t, time.Date(2025, time.February, 16, 0, 0, 0, 0, time.UTC))
```

Comment:
What's this about?

Reply:
These are temporary skips since there are some known issues with auto-destroy-duration ILM is patching. I've set an arbitrary date to fail these tests again if we forget to remove these skips once the fix is introduced.